app/src/main/java/com/iron/espresso/presentation/home/mystudy/MyStudyDetailActivity.kt

```diff
@@ -27,9 +29,8 @@ class MyStudyDetailActivity :
     BaseActivity<ActivityMystudyDetailBinding>(R.layout.activity_mystudy_detail) {
 
     private val viewModel by viewModels<MyStudyDetailViewModel>()
-    private var authority = ""
     private var studyId = -1
-    private var participateList = arrayListOf<ParticipateItem>()
+    private var studyInfoItem = StudyInfoItem()
```

Comment:
```suggestion
    private var studyInfoItem: StudyInfoItem? = null
```
빈 객체를 만들어 두는 이유가 있을까요?
nullable 객체를 사용하는게 적합한 케이스 같아요

Reply:
@wswon cb54150
수정하였습니다.